CHANGELOG.md

```diff
@@ -133,8 +133,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fixed bug for searchable snapshot to take 'base_path' of blob into account ([#6558](https://github.com/opensearch-project/OpenSearch/pull/6558))
 - Fix fuzziness validation ([#5805](https://github.com/opensearch-project/OpenSearch/pull/5805))
 - Fix GetSnapshots to not return non-existent snapshots with ignore_unavailable=true ([#6839](https://github.com/opensearch-project/OpenSearch/pull/6839))
+- Resolve windows opensearch-env.bat having variable delayed expansion issues on OPENSEARCH_HOME assignment in [#6956](https://github.com/opensearch-project/OpenSearch/pull/6956) ([#7080](https://github.com/opensearch-project/OpenSearch/pull/7080))
```

Comment:
Since this is a fix for an unreleased change, then we probably don't need a changelog entry and you can remove this. The entry "Changed `opensearch-env` to respect already set `OPENSEARCH_HOME` environment variable" is really all readers of the changelog will care about upon the next release.

Reply:
Is it possible to overwrite the changelog check? That is, make it so it does not complain if you do not have a difference in the changelog?

Reply:
Hi @andrross I have resolved the changelog comment you mentioned.
Please review again.
Thanks!